.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,27 @@
 #### What type of this PR
 
 Add one of the following kinds:
-/kind bug
-/kind cleanup
-/kind documentation
 /kind feature
-/kind design
+/kind bug
+/kind polish
+/kind document
 
 #### What this PR does / why we need it:
 
-
 #### Which issue(s) this PR fixes:
-Fixes #
+
+- Fixes #your-issue_number
+- [Erda Cloud Issue Link](paste your link here)
 
 #### Specified Reviewers:
+
 /assign @your-reviewer
 
+#### Need cherry-pick to release versions?
+
+- [ ] YES
+- [x] NO
```

Comment:
it's bad, it will make PR have a 1/2 task bar

Reply:
you are right.